src/backend/marsha/core/defaults.py

```diff
@@ -58,3 +58,27 @@
 # FLAGS
 SENTRY = "sentry"
 BBB = "BBB"
+
+# Emails sent for scheduled webinars
+REMINDER_1 = "REMINDER_1"
+REMINDER_2 = "REMINDER_2"
+REMINDER_3 = "REMINDER_3"
+REMINDER_IS_STARTED = "REMINDER_IS_STARTED"
+
+REMINDER_FROM_MS = "FROM_MS"
+REMINDER_REGISTER_BEFORE_MS = "REGISTER_BEFORE_MS"
+
+REMINDERS_STEP = {
+    REMINDER_1: {
+        REMINDER_FROM_MS: 5 * 60 * 1000,  # 5 minutes before
+        REMINDER_REGISTER_BEFORE_MS: 3 * 60 * 60 * 1000,  # three hours before
+    },
+    REMINDER_2: {
+        REMINDER_FROM_MS: 3 * 60 * 60 * 1000,  # three hours before
+        REMINDER_REGISTER_BEFORE_MS: 1 * 24 * 60 * 60 * 1000,  # 1 day before
+    },
+    REMINDER_3: {
+        REMINDER_FROM_MS: 3 * 24 * 60 * 60 * 1000,  # three days before
+        REMINDER_REGISTER_BEFORE_MS: 30 * 24 * 60 * 60 * 1000,  # thirty days before
+    },
+}
```

Comment:
Maybe all of this should be in the settings ? Doing this allow us to add or remove a reminder or update its configuration easily